pkg/hubble/parser/threefour/parser.go

```diff
@@ -306,20 +307,24 @@ func decodeEthernet(ethernet *layers.Ethernet) *pb.Ethernet {
 	}
 }
 
-func decodeIPv4(ipv4 *layers.IPv4) (ip *pb.IP, src, dst net.IP) {
+func decodeIPv4(ipv4 *layers.IPv4) (ip *pb.IP, src, dst netip.Addr) {
+	src, _ = ippkg.AddrFromIP(ipv4.SrcIP)
```

Comment:
Same here and the rest, we should probably explain why we're ignoring errors in a comment.